dl/pom.xml

```diff
@@ -239,4 +240,27 @@
             </plugin>
         </plugins>
     </reporting>
+    
+    <profiles>
+    	<profile>
+    		<id>parallel-tests</id>
+   			<properties>
+      			<runSpecsInParallel>true</runSpecsInParallel>
+      			<tagsToExclude>com.intel.analytics.bigdl.tags.Serial</tagsToExclude>
+   			</properties>
+		</profile>
+		<profile>
+    		<id>serial-tests</id>
+   			<properties>
+      			<runSpecsInParallel>false</runSpecsInParallel>
+      			<tagsToExclude>com.intel.analytics.bigdl.tags.Parallel</tagsToExclude>
+   			</properties>
+		</profile>
+		<profile>
+    		<id>serial-all</id>
+   			<properties>
+      			<runSpecsInParallel>false</runSpecsInParallel>
+   			</properties>
+		</profile>
+    </profiles>
```

Comment:
it is impossible to run both test profiles in one maven comamnd. see the discussion here:　http://maven.40175.n5.nabble.com/Multiple-profile-execution-in-maven-td2700693.html
we have to run the two profiles in two commands.